src/app/admin/subjects/subjects.component.ts

```diff
@@ -39,10 +55,52 @@ export class SubjectsComponent implements OnInit {
           return of(null);
         })
       )
-      .subscribe((newData: SSubjects | null) => {
+      .subscribe((newData: Array<ISubjects> | null) => {
+        if (newData) {
+          this.subjectTableList = [...this.subjectTableList, ...newData];
```

Comment:
seems like this.subjectTableList.push(newData) can be used

Reply:
Alredy fixed this in a new wersion. I use just a showSubjects function to show new resuts